content/syllabuses/short-course-devops-foundational.md

```diff
@@ -0,0 +1,80 @@
+---
+title: DevOps Short Course – Foundational
+---
+
+Welcome to the DevOps Short Course, the foundational level.
+
+This course is to be completed in 6 weeks.
+
+Learners should spend 4 to 6 hours every week on self paced learning, including reading, working on tasks, quizzes etc.
+
+## General information
+
+### Group session structure
+We will have two 1h group sessions per week. They will mostly follow the structure below:
+
+#### Session 1
+- Review tasks and questions from last week
+- Workshop: the facilitator will present concepts, explain the content, ask questions, give answers, open a dialogue with the learners, demonstrate practical concepts
+- Task and homework: give the learners a task to do during the session, to be finished at home if needed and what tasks should be done at home
+
+#### Session 2
+- Workshop: same as above
+- Task and homework: same as above
+
+The week's tasks and quizzes must be sent by latest on Sunday.
+
+### How to submit answers to the conceptual questions
+Through out the course we will have conceptual questions requiring written answers. Follow the the instructions below to submit them:
+
+{{< contentlink path="project-submission-instructions/markdown-questions" >}}.
+
+## Course Content
+
+### Week 0: Introduction to Umuzi
+Here you will learn about tools and practices used through out an Umuzi short course.
+
+First, take the [Intro to Tilde for non-coder bootcampers]({{< ref "syllabuses/tilde-intro-non-tech-student" >}}).
```

Comment:
I am using the DS short course as reference.

There, these Tilde/Git/GitHub pieces of content are referenced on the Syllabus, so I thought they would be loaded on the learners Tilde board.

My question is: if I add a `contentlink` reference on the `content/syllabuses/short-course-devops-foundational.md` file, will it appear as a card in the Tilde board? Or only the references from the `content/devops/` files?

Reply:
Any contentlink that points to a project or topic will appear as a card.
If a learner needs to do another course (like the intro to Tilde course) then it means we need to register them for that course on the backend. 

I would like to upgrade things at some point so that one course can simply include another. 

On the DS short course, I included the intro to Tilde for non-techies first because I wanted to give the learners a chance to dig into datascience concepts without first needing to learn about git. Those guys might not be very technical yet so hitting them with git up front might have given them a bad experience. So things are split up there so that we can lead them in gently. 

So, even with that course, it'll be important for the person setting up the learners to make sure everyone has the right courses loaded up.

Reply:
Got it.

Can you register the "Intro to Tilde for non-coder bootcampers" for them on the backend?

Also, I'll keep the other two projects listed because, by my survey, most of the learners know Git/GitHub but never used them or don't even know what they are.

Since the first set of conceptual questions should be sent via a git repository, they will need to know how to use Git/GitHub ASAP.

I think it makes sense to leave things as is.

What do you think?